themes/vue/source/css/_scrimba.styl

```diff
@@ -25,7 +28,7 @@
       width 0
       height 0
       top 5px
-      left 8px
+      right 6px
```

Comment:
Why reducing 2px?

Reply:
The white play button is not exactly in the center of the blue circle it has 12px left margin and 10px right margin, so now that we define the space from the right.